src/api.ts

```diff
@@ -152,8 +219,11 @@ export class HomebridgeAPI extends EventEmitter implements API {
     super();
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static isLegacyPlatformPlugin(platformPlugin: any): platformPlugin is LegacyPlatformPlugin {
+  public static isDynamicPlatformPlugin(platformPlugin: PlatformPlugin): platformPlugin is DynamicPlatformPlugin {
+    return "configureAccessory" in platformPlugin;
+  }
+
+  public static isLegacyPlatformPlugin(platformPlugin: PlatformPlugin): platformPlugin is StaticPlatformPlugin {
```

Comment:
should this be renamed to `isStaticPlatformPlugin` as well?

Reply:
oh yes, missed that, thanks